client/src/api/datasource/ApiClient.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import { DataSourceClient } from './DataSourceClient';
+
+export class ApiClient implements DataSourceClient {
+  list() {
+    return axios.get('/api/datasource');
```

Comment:
URL is /api/datasources

The plural is used across all the api

Reply:
There will need to be changes on this for the actual implementaiton this are just stubs in place for the moment